### PR TITLE
Spotlight Reviews

### PR DESCRIPTION
**Pull request by Raemon**
This PR
– makes it so SpotlightItems will show "review comments" for a given post, if there are any above 10 karma for that post. (i.e. if a spotlight was a LessWrong Review Winner, people who wrote good reviews will also be immortalized). It shows at most 2 reviews

– puts ai safety ReviewWinners on the Alignment Forum, where they rotate through once a day.

image

image

image

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1208863157897323) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by Raemon** [comment]
I updated it so we get some nice translucent background for SingleLineItems (but, get an opaque white background for expanded comments, to avoid some UI craziness)

image

**Comment by Raemon** [comment]
I have tested on mobile and darkmode. On mobile ("xs" or smaller) we don't show the reviews at all. Added some darkmode translucent-background options (which were previously missing and probably hurting us somewhere)

File: packages/lesswrong/lib/collections/posts/schema.tsx [hunk 1]
```diff
@@ -3111,6 +3111,23 @@ const schema: SchemaType<"Posts"> = {
...
+    canRead: ['guests'],
+    graphqlArguments: 'commentsLimit: Int, maxAgeHours: Int, af: Boolean',
+    // commentsLimit for some reason can receive a null (which was happening in one case)
+    // we haven't figured out why yet
+    resolver: async (post: DbPost, args: {}, context: ResolverContext) => {
+      const { currentUser, Comments } = context;
+      const reviews = await context.Comments.find({postId: post._id, baseScore: {$gte: 10}, reviewingForReview: {$ne: null}}, {sort: {baseScore: -1}, limit: 2}).fetch();
+      return await accessFilterMultiple(currentUser, Comments, reviews, context);
+    }
+  }),
```
Comment (b0b3rt) [inline packages/lesswrong/lib/collections/posts/schema.tsx:3126]: This should ideally have a corresponding sqlResolver

File: packages/lesswrong/lib/collections/posts/schema.tsx [hunk 2]
```diff
@@ -3111,6 +3111,23 @@ const schema: SchemaType<"Posts"> = {
     canUpdate: [userOwns, "admins"],
     ...schemaDefaultValue(false)
   },
+  reviews: resolverOnlyField({
+    type: Array,
+    graphQLtype: "[Comment]",
+    canRead: ['guests'],
+    graphqlArguments: 'commentsLimit: Int, maxAgeHours: Int, af: Boolean',
```
Comment (b0b3rt) [inline packages/lesswrong/lib/collections/posts/schema.tsx:3118]: Why is this accepting `graphqlArguments` if it's not using any of them

**Review by b0b3rt — APPROVED** [review]

**Pull request merged**